storybookjs/storybook
# Web-components: Fix cleaning Lit Expression comments

We are trying to create a design guideline with lit and want to use storybook to document our components. Here is a example story (in mdx format) that I use:

```jsx
import { html } from 'lit';
import { Canvas, Story } from '@storybook/addon-docs';
import './gr-button';

export const Template = (args) => html`<gr-button 
    ?secondary=${args.variant === 'secondary'} 
    ?disabled=${args.disabled} 
    ?block=${args.block} 
    ?medium=${args.size === 'medium'} 
    ?small=${args.size === 'small'}>${args.label}</gr-button>`;

<Canvas columns={1}>
  <Story name="Secondary Block disabled button"  args={{ disabled: true, variant: 'secondary', block: true, label: 'Fill the form' }}>
    {Template.bind({})}
  </Story>
</Canvas>
```

And here the result that Storybook shows in "Show code" panel in the UI:

```html
<gr-button secondary="" disabled="" block="">Fill the form</gr-button>
```

This fix cleans Lit expression comments from output (``)

## What I did

There was a regexp replace for cleaning comment tags, I just updated this regexp by taking the one from lit-html repository.

## How to test

There was no test case related with regexp replace part in current code.  I wanted to add extra test cases for this but I'm very confused while I was trying to write it. I'll check it again but creating this draft anyway to be able to share what I'm doing for solving this issue. 

I also think updating kitchen sink app with an example that has the condition above will be helpful.
